app/src/main/java/net/skyscanner/backpack/demo/compose/Text.kt

```diff
@@ -21,39 +21,117 @@ package net.skyscanner.backpack.demo.compose
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import net.skyscanner.backpack.compose.text.BpkText
 import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkDimension
 
+@Preview
 @Composable
-fun TypographyStylesStory() {
-  val styles = listOf(
-    BpkTheme.typography.hero1 to "Hero 1",
-    BpkTheme.typography.hero2 to "Hero 2",
-    BpkTheme.typography.hero3 to "Hero 3",
-    BpkTheme.typography.hero4 to "Hero 4",
-    BpkTheme.typography.hero5 to "Hero 5",
-    BpkTheme.typography.heading1 to "Heading 1",
-    BpkTheme.typography.heading2 to "Heading 2",
-    BpkTheme.typography.heading3 to "Heading 3",
-    BpkTheme.typography.heading4 to "Heading 4",
-    BpkTheme.typography.heading5 to "Heading 5",
-    BpkTheme.typography.subheading to "Subheading",
-    BpkTheme.typography.bodyLongform to "Body Longform",
-    BpkTheme.typography.bodyDefault to "Body Default",
-    BpkTheme.typography.footnote to "Footnote",
-    BpkTheme.typography.caption to "Caption",
-    BpkTheme.typography.label1 to "Label 1",
-    BpkTheme.typography.label2 to "Label 2",
-  )
-  LazyColumn(modifier = Modifier.padding(16.dp)) {
+fun HeroStyleStory() {
+  BackpackPreview {
+    val styles = listOf(
+      BpkTheme.typography.hero1 to "Hero 1",
+      BpkTheme.typography.hero2 to "Hero 2",
+      BpkTheme.typography.hero3 to "Hero 3",
+      BpkTheme.typography.hero4 to "Hero 4",
+      BpkTheme.typography.hero5 to "Hero 5",
+    )
+    TypographyStylesStory(styles = styles)
+  }
+}
+
+@Preview
+@Composable
+fun HeadingStyleStory() {
+  BackpackPreview {
+    val styles = listOf(
+      BpkTheme.typography.heading1 to "Heading 1",
+      BpkTheme.typography.heading2 to "Heading 2",
+      BpkTheme.typography.heading3 to "Heading 3",
+      BpkTheme.typography.heading4 to "Heading 4",
+      BpkTheme.typography.heading5 to "Heading 5",
+    )
+    TypographyStylesStory(styles = styles)
+  }
+}
+
+@Preview
+@Composable
+fun BodyStyleStory() {
+  BackpackPreview {
+    val styles = listOf(
+      BpkTheme.typography.subheading to "Subheading",
+      BpkTheme.typography.bodyLongform to "Body Longform",
+      BpkTheme.typography.bodyDefault to "Body Default",
+      BpkTheme.typography.footnote to "Footnote",
+      BpkTheme.typography.caption to "Caption",
+      BpkTheme.typography.label1 to "Label 1",
+      BpkTheme.typography.label2 to "Label 2",
+    )
+    TypographyStylesStory(styles = styles)
+  }
+}
+
+@Composable
+private fun TypographyStylesStory(styles: List<Pair<TextStyle, String>>) {
+  LazyColumn(modifier = Modifier.padding(BpkDimension.Spacing.Base)) {
     items(styles) { style ->
-      Text(
+      BpkText(
         style = style.first,
         text = style.second
       )
     }
   }
 }
+
+@Preview
+@Composable
+fun DefaultText() {
```

Comment:
Add we add Preview or Story suffix for this cases?

Reply:
I used `Example` as that felt more "correct" - i.e. we're previewing an example of a default text component